content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,384 @@
+
+.. image:: maintenance_calendar/maintenance-calendar-favorites-popover.png
+   :align: center
+   :alt: Favorites section of filters drop-down menu.
+
+To save a new :guilabel:`Favorite` search, select the desired :guilabel:`Filters`. Then, click
+:guilabel:`Save current search`. In the field directly below :guilabel:`Save current search`, assign
+a name to the search.
+
+Under the assigned name, there are two options, to save the current search either as the
+:guilabel:`Default filter`, or as a :guilabel:`Shared` filter.
+
+Selecting :guilabel:`Default filter` sets this filter as the default when opening this calendar
+view.
+
+Selecting the :guilabel:`Shared` filter makes this filter available to other users.
+
+Once ready, click :guilabel:`Save`. When clicked, the new :guilabel:`Favorite` filter appears
+in the :guilabel:`Favorites` column, and a :guilabel:`⭐ (gold star)` icon appears with the filter's
+name in the search bar.
+
+Views
+-----
+
+The :guilabel:`Maintenance Calendar` is available in six different views: :guilabel:`Calendar`
+(default), :guilabel:`Kanban`, :guilabel:`List`, :guilabel:`Pivot`, :guilabel:`Graph`, and
+:guilabel:`Activity`.
+
+.. image:: maintenance_calendar/maintenance-calendar-view-type-icons.png
+   :align: center
+   :alt: Different view type icons for maintenance calendar.
+
+Calendar view
+~~~~~~~~~~~~~
+
+:guilabel:`Calendar` is the default view displayed when the :guilabel:`Maintenance Calendar` is
+opened. There are a number of options in this view type for sorting and grouping information about
+maintenance requests.
+
+In the top-left corner of the page, there is a drop-down menu set to :guilabel:`Week`, by default.
+Clicking that drop-down menu reveals the different periods of time, in which the calendar can be
+viewed: :guilabel:`Day`, :guilabel:`Month`, and :guilabel:`Year`. There is also an option to
+:guilabel:`Show weekends`, selected by default. If unselected, weekends are not shown on the
+calendar.
+
+.. image:: maintenance_calendar/maintenance-calendar-period-dropdown.png
+   :align: center
+   :alt: Calendar period drop-down menu options.
+
+To the left of this menu, there is a :guilabel:`⬅️ (left arrow)` icon and a :guilabel:`➡️ (right
+arrow)` icon. Clicking these arrows moves the calendar backward or forward in time, respectively.
+
+To the right of the drop-down menu set to :guilabel:`Week`, by default, is a :guilabel:`Today`
+button. Clicking this button resets the calendar to view today's date, no matter which point in time
+is being viewed before clicking it.
+
+At the far-right side of the page is a sidebar column, containing a minimized calendar set to
+today's date, and a :guilabel:`Technician` list, displaying all the *Technicians* with requests
+currently open. Click the :guilabel:`(panel)` icon at the top of this sidebar to open or close the
+sidebar.
+
+.. image:: maintenance_calendar/maintenance-calendar-collapsible-panel.png
```

Comment:
this is a pretty big screenshot, consider resizing to minimum viability or removing.